src/ServiceProvider.php

```diff
@@ -50,6 +50,16 @@ function (string $feature) {
                 return !$unleash->isFeatureEnabled($feature);
             }
         );
+
+        Blade::directive(
+            'featuresList',
+            function () {
+                $client = app(Client::class);
+                $unleash = app(Unleash::class, ['client' => $client]);
+
+                return $unleash->getAvailableFeaturesCollection();
+            }
+        );
```

Comment:
I feel like this may be a very specific use case. Does this return an array? If so you could just use the main `Unleash` class to retrieve this so I'm not sure how necessary a Blade directive is here.

Reply:
Yes, this is indeed a specific use case. 

The unleash method return a collection. 
Collection {#297 ▼
  #items: array:3 [▼
    "smartbar_added" => false
    "sub_site_title" => true
    "test_user_ids" => true
  ]
}
In blade it is shown as the json version.
{"smartbar_added":false,"sub_site_title":true,"test_user_ids":true}

We use this in de <script> block so javascript knows directly which feature is enabled.

What is your opinion? How do you handle feature toggle for javascript?

Reply:
Ah I see. I would maybe use the JS SDK for this if I were going to do this but can appreciate if Laravel is your stack then this would be really useful.

As it's a very specific use case I would like to avoid having to maintain this so maybe this is one to keep in your own project. Or alternatively submit as a separate PR, if we can get enough interest then we can reconsider it.

Reply:
I see your point. This is indeed ok to be in our own project. It is removed by the last new commits.

Thanks for suggesting the JS SDK. Our frontender has doubts about revealing our gitlab url with javascript, that's why Laravel did this by the way for it. But we will reconsider it.